dataframely/columns/_base.py

```diff
@@ -97,7 +102,25 @@ def validation_rules(self, expr: pl.Expr) -> dict[str, pl.Expr]:
         if not self.nullable:
             result["nullability"] = expr.is_not_null()
         if self.check is not None:
-            result["check"] = self.check(expr)
+            if isinstance(self.check, dict):
+                for k, v in self.check.items():
+                    result[k] = v(expr)
```

Comment:
And I was wondering whether we should prefix every check with `check_` to avoid name collisions.

Reply:
I still want to have the `check_` prefix to ensure that the user cannot provide a name that is already used by other validation rules